apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -4282,6 +4282,117 @@
       }
     }
   },
+  "sui": {
+    "account": {
+      "stakedBalance": "Delegated asset",
+      "stakedBalanceTooltip": "This amount is currently staked with validators.",
+      "availableBalance": "Withdawable balance",
+      "availableBalanceTooltip": "This amount has been unstaked and is currently able to be withdrawn.",
+      "pendingBalance": "Pending balance",
+      "pendingBalanceTooltip": "This amount has been unstaked and is currently pending availability.",
+      "storageUsageBalance": "Minimum balance",
+      "storageUsageBalanceTooltip": "This amount is reserved for storage and transactions, and is the minimum balance that your account must maintain to remain active.",
+      "subHeader": {
+        "cardTitle": "Powered by Figment",
+        "moreInfo": "More info",
+        "drawerTitle": "Powered by Figment",
+        "title": "The SUI token is now available on Ledger Live",
+        "description": "You can now start to manage your SUI tokens and secure them through Ledger Live.",
+        "description2": "The SUI token is native to the SUI chain and is used for everything from transactions, staking and many upcoming features.",
+        "website": "Powered by Figment"
+      }
+    },
+    "operationDetails": {
+      "extra": {
+        "stakedAmount": "Delegated amount"
+      }
+    },
+    "stake": {
+      "emptyState": {
+        "description": "You can earn Sui rewards by staking your assets.",
+        "info": "How Staking Works",
+        "earnRewards": "Earn rewards"
+      },
+      "table": {
+        "header": "Staking Position(s)",
+        "stake": "Add",
+        "staked": "Delegated",
+        "pending": "Pending Release",
+        "available": "Withdrawable"
+      },
+      "unstake": "Undelegate",
+      "activeTooltip": "Delegated amounts generate rewards",
+      "inactiveTooltip": "Delegated amounts do not generate rewards",
+      "commission": "Commission",
+      "totalStake": "Total Delegation",
+      "minSafeWarning": "Not enough funds",
+      "flow": {
+        "title": "Delegate",
+        "steps": {
+          "starter": {
+            "description": "You may earn rewards by staking your SUI assets to a validator.",
+            "bullet": [
+              "You keep ownership of delegated assets",
+              "Delegate using your Ledger device",
+              "Assets will be available 52 - 65 hours after unstaking"
```

Comment:
I believe this is not correct.

On SUI there's a lock-up period of 1 day when staking. During these 24h you can't unstake.
There is no lockup period when unstaking SUI, but you won't earn rewards for the last epoch.